src/main/java/frc/trigon/robot/subsystems/collector/simulationcollector/SimulationCollectorConstants.java

```diff
@@ -0,0 +1,80 @@
+package frc.trigon.robot.subsystems.collector.simulationcollector;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import frc.trigon.robot.motorsimulation.MotorSimulationConfiguration;
+import frc.trigon.robot.motorsimulation.SimpleMotorSimulation;
+
+public class SimulationCollectorConstants {
+    static final boolean
+            FOC_ENABLED = true;
+    private static final int
+            ANGLE_MOTOR_AMOUNT = 1,
+            COLLECTION_MOTOR_AMOUNT = 1;
+    private static final double
+            ANGLE_MOTOR_GEAR_RATION = 1,
+            COLLECTION_MOTOR_GEAR_RATIO = 1;
+    private static final double
+            ANGLE_MOTOR_MOMENT_OF_INERTIA = 0.0001,
+            COLLECTION_MOTOR_MOMENT_OF_INERTIA = 0.0001;
+    private static final DCMotor
+            ANGLE_GEARBOX = DCMotor.getKrakenX60Foc(ANGLE_MOTOR_AMOUNT),
+            COLLECTION_GEARBOX = DCMotor.getKrakenX60Foc(COLLECTION_MOTOR_AMOUNT);
+    private static final double
+            ANGLE_VOLTAGE_COMPENSATION_SATURATION = 12,
+            COLLECTION_VOLTAGE_COMPENSATION_SATURATION = 12;
+    private static final double
+            ANGLE_MOTOR_CONVERSION_FACTOR = 360,
+            COLLECTION_MOTOR_CONVERSION_FACTOR = 1;
+    static final SimpleMotorSimulation
+            ANGLE_MOTOR = new SimpleMotorSimulation(
+            ANGLE_GEARBOX,
+            ANGLE_MOTOR_GEAR_RATION,
+            ANGLE_MOTOR_MOMENT_OF_INERTIA
+    ),
+            COLLECTION_MOTOR = new SimpleMotorSimulation(
+                    COLLECTION_GEARBOX,
+                    COLLECTION_MOTOR_GEAR_RATIO,
+                    COLLECTION_MOTOR_MOMENT_OF_INERTIA
+            );
+    private static final double
+            ANGLE_P = 0,
+            ANGLE_I = 0,
+            ANGLE_D = 0,
+            ANGLE_KS = 0,
+            ANGLE_KV = 0,
+            ANGLE_KA = 0;
+    private static final double
+            ANGLE_MAX_VELOCITY = 0,
+            ANGLE_MAX_ACCELERATION = 0;
+
+    static {
+        configureAngleMotor();
+        configureCollectionMotor();
+    }
+
+    private static void configureAngleMotor() {
+        MotorSimulationConfiguration config = new MotorSimulationConfiguration();
+
+        config.pidConfigs.kP = ANGLE_P;
+        config.pidConfigs.kI = ANGLE_I;
+        config.pidConfigs.kD = ANGLE_D;
+
+        config.feedforwardConfigs.kS = ANGLE_KS;
+        config.feedforwardConfigs.kV = ANGLE_KV;
+        config.feedforwardConfigs.kA = ANGLE_KA;
```

Comment:
you should have a kg here

Reply:
But the ff calculation doesn't use the kg anyway

Reply:
For an arm simulation? Ofc it does

Reply:
But it's not an arm, it's a simpleMotorSimulation

Reply:
That's also a problem... Since this is an angular mechanism, this should be an arm sim